WordClock/Config.cpp

```diff
@@ -0,0 +1,33 @@
+#include <EEPROM.h>
+#include "Config.h"
+
+ConfigStorageStruct ClockConfig = {
+  CONFIG_VERSION,
+  "",
+  "",
+  8355711,
```

Comment:
Zouden we deze default values ook kunnen definiëren in Config.h?

Reply:
Een struct is een data type. Je geeft geen waardes aan een data type. Je geeft values aan instances/objects van het data type.

Reply:
Okay, ik dacht omdat het hier gaat om hardcoded values, dat we wellicht dat values ergens vast konden leggen.

Reply:
Aangepast